fe/src/components/Menu/Menu.re

```diff
@@ -1,31 +1,98 @@
-[%bs.raw {|require('./Menu.scss')|}];
 open EloTypes;
+open Svc;
+open Js.Promise;
 open BsReactstrap;
 
-let component = ReasonReact.statelessComponent("Menu");
+[%bs.raw {|require('./Menu.scss')|}];
+
+type state = {menuOpened: bool};
+type appSend = appContainerActions => unit;
+
+type action =
+  | ToggleMenu
+  | CloseMenu
+  | Logout(appSend);
+
+let component = ReasonReact.reducerComponent("Menu");
 
-let make = (~activePlace, _children) => {
+let onLogoutSuccess = appSend => {
+  appSend(SetIsLogged(false));
+  ReasonReact.Router.push("/");
```

Comment:
Odpaliłem sobie lokalnie brancha i po kliknięciu logout w konsoli jest warning, screen
![Zaznaczenie_001](https://user-images.githubusercontent.com/16646714/59974151-eaedad00-95a8-11e9-81f3-986985de6c70.png)
Mam podejrzenie że to przez to że robisz tutaj appSenda i przekierowanie, a ponieważ one są asynchroniczne to przekierowanie może się zrobić szybciej i wtedy appSend wysyła do komponentu który jest odmontowany. Ale to trzeba by sprawdzić. Jeśli moje podejrzenie byłoby prawdą to przekierowanie można robić w reducerze po odebraniu SetIsLogged

Reply:
Sprawdziłem na szybko i jak wywaliłem wogle to przekierowanie to wygląda że działa dobrze